assign1/assessGrade.cpp

```diff
@@ -10,65 +10,81 @@
 
 int assessGrade(char letterGrade[])
 {
-	char* input = letterGrade;
+	char str[MAX_INPUT] = "";
 
```

Comment:
why not do something like:
char first = '';
char second = '';
first = letterGrade[0]

if (strlen(letterGrade == 2){
     second = letterGrade[1]
}

...

if (first == 'A'){
   if (second == '-'){
    ...
    }
   else if (second == '+'){
     ...
  }
}